### PR TITLE
Add Tablo shard

### PR DESCRIPTION
**Pull request by hutou**
add [Tablo](https://github.com/hutou/tablo) shard : a flexible terminal table generator

### CODE REVIEW

**Comment by hutou** [comment]
Yes, as stated in the Readme, Tablo generates formatted text tables from 2D arrays.
What changes are requested ?

**Comment by veelenga** [comment]
Please revert your README autoformatting changes.

File: README.md [hunk 1]
```diff
@@ -150,6 +150,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
 ## CLI Utils
  * [cride](https://github.com/j8r/cride) - A light CLI text editor/IDE
  * [progress_bar.cr](https://github.com/TPei/progress_bar.cr) - A simple and customizable progress bar
+ * [tablo](https://github.com/hutou/tablo) - a flexible terminal table generator
```
Comment (veelenga) [inline README.md]:
```suggestion
 * [tablo](https://github.com/hutou/tablo) - A flexible terminal table generator
```

**Comment by hutou** [comment]
Ah, I misunderstood your remarks, but I just realized the problem now: the culprit is the source code beautifier I use, which changes asterisks into dashes! (see: Prettier https://github.com/arcticicestudio/styleguide-markdown/issues/3)
I just pushed the corrected version of readme.md.

**Pull request merged**